x/community/types/params.go

```diff
@@ -1,26 +1,39 @@
 package types
 
-import "time"
+import (
+	fmt "fmt"
+	"time"
+
+	sdkmath "cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
 
 var (
 	DefaultUpgradeTimeDisableInflation = time.Time{}
+	DefaultRewardsPerSecond            = sdk.NewCoin("ukava", sdkmath.ZeroInt())
```

Comment:
thoughts on setting this to a non-zero default? I think the default chain should mint here to align with mainnet behaviour.

Reply:
Updated to be target amount